scripts/globals/items/scroll_of_regen_v.lua

```diff
@@ -0,0 +1,16 @@
+-----------------------------------
+-- ID: 5086
+-- Scroll of Regen V
+-- Teaches the white magic Regen V
+-----------------------------------
+local item_object = {}
+
+item_object.onItemCheck = function(target)
+    return target:canLearnSpell(504)
+end
+
+item_object.onItemUse = function(target)
+    target:addSpell(504)
+end
```

Comment:
```suggestion
-----------------------------------
require("scripts/globals/status")
-----------------------------------
local item_object = {}

item_object.onItemCheck = function(target)
    return target:canLearnSpell(xi.magic.spell.REGEN_V)
end

item_object.onItemUse = function(target)
    target:addSpell(xi.magic.spell.REGEN_V)
end
```

It's been a long time since we've had new scrolls added, we have new enums to represent spells now 👍